koku/api/metrics/views.py

```diff
@@ -45,14 +48,25 @@ def get_queryset(self):
 
         Restricts the returned data to provider_uuid if supplied as a query parameter.
         """
-        queryset = CostModelMetricsMap.objects.all()
+        queryset = self.get_json(COST_MODEL_METRICS_FILE_NAME)
         source_type = self.request.query_params.get("source_type")
         if source_type:
-            queryset = queryset.filter(source_type=source_type)
+            # Query on source type
+            queryset = list(filter(lambda x: x.get("source_type") == source_type, queryset))
 
         return queryset
 
     @vary_on_headers(RH_IDENTITY_HEADER)
     def list(self, request, *args, **kwargs):
         """Obtain the list of CostModelMetrics for the tenant."""
         return super().list(request=request, args=args, kwargs=kwargs)
+
+    def get_json(self, path):
+        """Obtain API JSON data from file path."""
+        json_data = None
+        with open(path) as json_file:
+            try:
+                json_data = json.load(json_file)
+            except (IOError, json.JSONDecodeError) as exc:
+                LOG.exception(exc)
```

Comment:
We'll want to fail and provide the proper HTTP code if we can't load the JSON in. If that happens this API has nothing to return.

Reply:
resolved as per "so I'd probably swap this entire function with just a 1-line variable definition"